pkg/1password/cli.go

```diff
@@ -38,18 +45,27 @@ func SignIn(account string) (string, error) {
 
 	err := cmd.Run()
 	if err != nil {
+		var exitErr *exec.ExitError
+		if errors.As(err, &exitErr) {
+			l.Error(
+				"error executing command",
+				zap.Error(err),
+				zap.String("stderr", string(exitErr.Stderr)),
```

Comment:
Even if I deliberately invoke an error, stderr is always empty string in my testing. Is there something else I need to do to get data here?

Reply:
This case might be a little weird. It is possible that the 1password tool `op` doesn't actually write to stderr in case of an error. For most commands we are using `Output()` which returns stdout to you. In this case, we'd probably want to attach a reader to `cmd.Stdout` to capture the output to display it here. This specific change was a tack-on though, as the main change is to update `executeCommand()` below.

Reply:
To answer your question, maybe redirecting stdout to stderr will help?